app/js/lib/foxbox/services/motion-sensor.js

```diff
@@ -4,13 +4,68 @@ import BaseService from './base';
 
 const TYPE = 'motion-sensor';
 
+const p = Object.freeze({
+  onMotionStateChanged: Symbol('onMotionStateChanged'),
+});
+
+/**
+ * Converts motion state value to boolean. Considers unknown state (null) the
+ * same as state when motion is not detected.
+ *
+ * @param {Object} motionState Motion state object.
+ * @return {boolean}
+ * @private
+ */
+function motionStateToBoolean(motionState) {
```

Comment:
In `app/js/lib/foxbox/services.js`, a similar function is declared using `const`.
Let's use a consistent pattern in both cases.